README.md

```diff
@@ -85,7 +85,8 @@ Example command:
 
 ## 🔭 Future Features <a name="future-features"></a>
 
-
+- Add a class for person, student and Teacher
```

Comment:
- [ ] When clicking the link to your license the user gets a 404 error page not found. please make sure it works before requesting another review.